### PR TITLE
migrate velodrome

### LINKED ISSUE #2207: Velodrome TLS cert not auto-renewing
The **velodrome** (https://velodrome.istio.io) TLS cert is **not** auto-renewing and expires **12/19/19**.

image

### PR DESCRIPTION
**Pull request by clarketm**
fix #2207

- [x] Update A record for `velodrome`
- [x] Re-provisioned TLS cert

### CODE REVIEW

File: prow/Makefile
Comment (cjwagner) [inline prow/Makefile:33]: Make this autodeploy in a follow up?

**Review by cjwagner — APPROVED** [review]
/hold

File: prow/Makefile
Comment (clarketm) [reply prow/Makefile:33]: Will add it to [job](https://github.com/istio/test-infra/pull/2226/files#diff-fbde41226949bfb348b691c6d52c372cR25) as soon as #2226 merges (to avoid conflict).

**Pull request merged**
